libs/blocks/aside/aside.css

```diff
@@ -546,10 +796,37 @@
     margin-top: 0;
   }
 
+  .aside.split.bio .foreground.container .text .icon-area {
+    display: block;
+  }
+
   .aside.notification.small .foreground.container .text .icon-area {
     width: 40px;
     height: 40px;
   }
+  
+  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area {
+    height: 80px;
+  }
+
+  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area img{
```

Comment:
```suggestion
  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area img {
```

Reply:
Thanks @Sartxi I have added the missing space in the css.